schema.sql

```diff
@@ -51,4 +51,17 @@ CREATE TABLE visits(
     DATE DATE NOT NULL,
     CONSTRAINT fk_animals FOREIGN KEY(animals_id) REFERENCES animals(id),
     CONSTRAINT fk_vets FOREIGN KEY(vets_id) REFERENCES vets(id)
-);
\ No newline at end of file
+);
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+DROP TABLE visits;
+
+CREATE TABLE visits(
```

Comment:
- [ ] You should add indexes for FOREIGN KEYS in the schema.sql file. Adding indexes for FOREIGN KEYS or columns frequently used in queries is one of the basic solutions for performance problems. Please try it out. It will be something like this:

     ```sql
        CREATE INDEX ON visits (animal_id);
        CREATE INDEX ON visits (vet_id);
        CREATE INDEX ON owners (email);
     ```

Reply:
Oh Sorry, We have used CREATE INDEX and missed adding those to the schema.sql file.
Corrected now.

````
CREATE INDEX visits_animal_id ON visits(animal_id);
CREATE INDEX visits_vet_id ON visits(vet_id);
CREATE INDEX owners_email ON owners(email);
````

Thank You.